vectordatasource/transform.py

```diff
@@ -8150,7 +8152,13 @@ def tags_set_ne_min_max_zoom(ctx):
     for _, props, _ in layer['features']:
         min_zoom = props.pop('__ne_min_zoom', None)
         if min_zoom is not None:
-            props['min_zoom'] = min_zoom
+            # don't overstuff features into tiles when they are in the
+            # long tail of won't display, but make their min_zoom
+            # consistent with when they actually show in tiles
+            if ceil(min_zoom) == trunc(min_zoom) + 1:
```

Comment:
I think this is the same as "if min_zoom is an integer", in which case the `if/else` isn't necessary and we can just write `props['min_zoom'] = ceil(min_zoom)` for everything (because `ceil(x) = x` for integers).

Reply:
Something like: `props['min_zoom'] = ceil(min_zoom) if min_zoom % 1 > 0.5 else min_zoom`

Which basically says "if the fractional part is more than half round it up, otherwise keep it the same". If you prefer, it might be more readable as:

```python
if min_zoom % 1 > 0.5:
  min_zoom = ceil(min_zoom)
props['min_zoom'] = min_zoom
```

Reply:
Fix in https://github.com/tilezen/vector-datasource/pull/1734/commits/ec0584455fcc8fd17a6f5b4ffe0425bc6ab65d69.